doc/nrf/ug_nrf5340.rst

```diff
@@ -630,14 +630,20 @@ This container package can be used by update tools to pass both images during th
 Getting logging output
 **********************
 
-When connected to a computer, the nRF5340 DK emulates three virtual COM ports.
+When connected to a computer, the nRF5340 DK emulates two virtual COM ports.
```

Comment:
I don't like that we say two COM ports, when it is not that simple.

Could we say that the v2.0.0 nRF5340 DK emulates two virtual COM ports ?